src/GitTfs.Vs2022/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("GitTfs.Vs2022")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("4dfaf7d4-7ae1-49e9-a58c-aa4ffe5a6d68")]
```

Comment:
This should be a different GUID from the Vs2019 assembly.

Reply:
Fixed